app/src/main/res/layout/view_user_profile.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="@dimen/profile_padding_vertical_default"
+    android:paddingStart="@dimen/profile_padding_horizontal_default"
+    android:paddingEnd="@dimen/profile_padding_horizontal_default"
+    android:paddingBottom="@dimen/profile_padding_vertical_default"
+    android:dividerPadding="@dimen/profile_padding_divider" >
+
+    <Spinner
+        android:id="@+id/sex_spinner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/profile_sex_prompt"/>
```

Comment:
Can we add some padding and/or flavor text above this? It's kinda hard to press the spinner otherwise.